core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.html

```diff
@@ -286,4 +286,19 @@
       font-size: 2vw;
     }
   }
+
+  @media only screen and (max-width: 420px) {
+    .dropdown-label {
+      font-size: 18px;
+      margin-right: 2vw;
+      /* This 'important' label is necessary to match the width
+      of the topic selector dropdown on smaller screen sizes. */
```

Comment:
I think this doesn't really explain why is the `!important` needed and why it cannot be achieved any other way.